packages/rest/test/acceptance/bootstrapping/application.acceptance.ts

```diff
@@ -61,12 +66,14 @@ describe('Bootstrapping the application', () => {
       }
 
       class MyComponent {
-        controllers = [MyController];
+        controllers: Array<Constructor<{}>> = [MyController];
```

Comment:
Is this type necessary? Shouldn't TypeScript figure this out on its own?

I think we should be using `ControllerClass[]` instead of `Array<...>` if the explicit type signature is required.